### PR TITLE
Added create_tables.sql

### PR DESCRIPTION
**Pull request by Svitoij**

### CODE REVIEW

File: src/main/resources/create_tables.sql [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    description TEXT,
+    salary DECIMAL(10, 2),
+);
+
+create table company (
+    id BIGSERIAL NOT NULL PRIMARY KEY,
+    name VARCHAR(100) NOT NULL
+);
+
+create table response (
```
Comment (wikigreen) [inline src/main/resources/create_tables.sql]: What does this table name mean?

File: src/main/resources/create_tables.sql [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    date_of_birth DATE NOT NULL,
+    gender VARCHAR(7) NOT NULL,
+    email VARCHAR(100),
+    status roles,
+    UNIQUE(email)
+);
+
+CREATE TABLE vacancies (
+    id BIGSERIAL NOT NULL PRIMARY KEY,
+    vac_name VARCHAR(50),
```
Comment (wikigreen) [inline src/main/resources/create_tables.sql]: Can be simplified to
```suggestion
    name VARCHAR(50),
```

File: src/main/resources/create_tables.sql [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    salary DECIMAL(10, 2),
+);
+
+create table company (
+    id BIGSERIAL NOT NULL PRIMARY KEY,
+    name VARCHAR(100) NOT NULL
+);
+
+create table response (
+    id BIGSERIAL NOT NULL PRIMARY KEY,
```
Comment (wikigreen) [inline src/main/resources/create_tables.sql]: If this is relation then id is not needed

File: src/main/resources/create_tables.sql [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    user_id BIGINT REFERENCES users(id),
+    vacancy_id BIGINT REFERENCES vacancies(id)
+);
+
+create table search (
+    id BIGSERIAL NOT NULL PRIMARY KEY,
```
Comment (wikigreen) [inline src/main/resources/create_tables.sql]: Same as previous

File: src/main/resources/create_tables.sql [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    user_id BIGINT REFERENCES users(id),
+    vacancy_id BIGINT REFERENCES vacancies(id)
+);
```
Comment (wikigreen) [inline src/main/resources/create_tables.sql]:
```suggestion
);

```
https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log

**Review by wikigreen — CHANGES_REQUESTED** [review]

File: src/main/resources/create_tables.sql (hunk 1 above)
Comment (Svitoij) [reply src/main/resources/create_tables.sql]: The name of this table means the ratio of the user to the number of vacancies to which he/she responded "response".

File: src/main/resources/create_tables.sql (hunk 4 above)
Comment (Svitoij) [reply src/main/resources/create_tables.sql]: Everything is ready for the next review

File: src/main/resources/create_tables.sql (hunk 5 above)
Comment (Svitoij) [reply src/main/resources/create_tables.sql]: Everything is ready for the next review

File: src/main/resources/create_tables.sql (hunk 2 above)
Comment (Svitoij) [reply src/main/resources/create_tables.sql]: corrected

File: src/main/resources/create_tables.sql (hunk 5 above)
Comment (wikigreen) [reply src/main/resources/create_tables.sql]: Did you understand what this comment is about? Did you check the article?

File: src/main/resources/create_tables.sql (hunk 1 above)
Comment (wikigreen) [reply src/main/resources/create_tables.sql]: Commonly such tables where we specify a relation between tables `x` and `y` we should name them "x_y".

File: src/main/resources/create_tables.sql [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+    id BIGSERIAL NOT NULL PRIMARY KEY,
+    name VARCHAR(100) NOT NULL
+);
+
+create table response (
+    user_id BIGINT REFERENCES users(id),
+    vacancy_id BIGINT REFERENCES vacancies(id)
+);
+
+create table search (
```
Comment (wikigreen) [inline src/main/resources/create_tables.sql]: Commonly such tables where we specify a relation between tables `x` and `y` we should name them "x_y".

**Review by wikigreen — CHANGES_REQUESTED** [review]

File: src/main/resources/create_tables.sql (hunk 5 above)
Comment (Svitoij) [reply src/main/resources/create_tables.sql]: Yes, fixed

File: src/main/resources/create_tables.sql (hunk 6 above)
Comment (Svitoij) [reply src/main/resources/create_tables.sql]: I'm ready for a new review

**Review by wikigreen — APPROVED** [review]
LGTM

**Pull request merged**